README.md

```diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
 <script src="https://unpkg.com/buefy/dist/buefy.min.js"></script>
 ```
 
+
+### Installing Developer Snapshots from GitHub npm Registry to Access the Latest Features
+To incorporate the latest changes from the development branch of buefy-next, you can install developer snapshots from the GitHub npm registry. However, proceed with caution as these packages may be deleted or retracted without notice.
+
+Steps:
+1. Generate a Personal Access Token:
+    - Create a personal access token on GitHub with read access to the GitHub Packages registry.
+
+2. Configure npm:
+    - In your terminal, execute the following commands:
+```bash
+npm login --auth-type=legacy --registry=https://npm.pkg.github.com
+ USERNAME: <their GitHub username>
+ PASSWORD: <personal access token issued at Step 1>
+echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
+```
```

Comment:
It would look better structured if you indent the code snippet and remove the itemization of the sentence "In your terminal...":

```
2. Configure npm:
    
   In your terminal, execute the following commands:

    ```bash
    npm login --auth-type=legacy --registry=https://npm.pkg.github.com
     USERNAME: <their GitHub username>
     PASSWORD: <personal access token issued at Step 1>
    echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
    ```
```

2. Configure npm:
    
    In your terminal, execute the following commands:

    ```bash
    npm login --auth-type=legacy --registry=https://npm.pkg.github.com
     USERNAME: <their GitHub username>
     PASSWORD: <personal access token issued at Step 1>
    echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
    ```

vs

2. Configure npm:
    - In your terminal, execute the following commands:
```bash
npm login --auth-type=legacy --registry=https://npm.pkg.github.com
 USERNAME: <their GitHub username>
 PASSWORD: <personal access token issued at Step 1>
echo "@ntohq:registry=https://npm.pkg.github.com" > .npmrc
```